### PR TITLE
Add releaser

### PR DESCRIPTION
**Pull request by katcipis**
Shifting from Full manual to Semi Auto =P. Not fully auto yet.

### CODE REVIEW

File: hack/releaser.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	print("usage: %s <version>\n\n", $ARGS[0])
+	exit("1")
+}
+
+version        = $ARGS[1]
+supported_os   = ("linux" "darwin" "windows")
+supported_arch = ("amd64")
+
+# Guarantee passing tests at least on the host arch/os
+make test
```
Comment (tiago4orion) [inline hack/releaser.sh]: nice =)

**Comment by tiago4orion** [comment]
codecov is freaking out... No go files were touched.

**Comment by tiago4orion** [comment]
sorry :( closed by mistake hahaha

**Comment by katcipis** [comment]
auheuheuaheuaheuaheauh, will merge soon if @vitorarins does not make any movements

**Comment by vitorarins** [comment]
We can add this: https://docs.travis-ci.com/user/deployment/releases/

in a later PR maybe

**Pull request merged**